source/utilities.cc

```diff
@@ -938,12 +948,166 @@ namespace aspect
                                      const MPI_Comm &comm)
     {
       std::string data_string;
+      std::stringstream urlString;
 
       if (Utilities::MPI::this_mpi_process(comm) == 0)
         {
           // set file size to an invalid size (signaling an error if we can not read it)
           unsigned int filesize = numbers::invalid_unsigned_int;
 
+          /**
+           * Lookup and use data from the provided url
+           * Author: Kodi Neumiller
+           * Added: 10/9/18
+           **/
+          //----Only run if the user wishes to use the libdap packages----//
```

Comment:
please remove this comment line

Reply:
Did you want me to remove the comment line "//----Only run if the user wishes to use the libdap packages----//"?

Reply:
yes